README.md

```diff
@@ -97,7 +97,7 @@ and uninstalled through
 
     $ sudo make uninstall.comp
```

Comment:
With the proposed Makefile, our examples for install and uninstall should include `PREFIX=/usr`.
We can safely assume very few would be interested in reading their shell's manual to install scripts that come with instructions to install.